docs/gettingStarted/customScheme/developCustomNonUniScheme.md

```diff
@@ -0,0 +1,94 @@
+  *Tutorial for adding non-universal scheme* [skip](../developCustomUniScheme)
+
+## Design Principle
+  TODO: Need Oren's input here
+
+## Setup
+
+- We will use *Alchemy-starter* for this tutorial, enter the directory and install starter-package
+  
+        cd alchemy-starter/
+        npm i
+
+- Add your custom scheme contract to `contracts` folder. Refer [*BuyInWithRageQuitOpt.sol*](#example-buyinwithragequitoptsol)
+
+- Compile your contracts
+
+        npm run compile
+  
+## Example: *BuyInWithRageQuitOpt.sol*
+
+Here is a simple non-universal scheme that allow People to Buy reputation by donating money to the DAO and if their goals no more align with the DAO, have the ability to Quit at some later time by giving up reputation
+
```

Comment:
how about just give a link to this contract ?

Reply:
Yeah can give link to contract in `alchemy-starter` example.

But this might not work for the other code-samples (for Subgraph, Client or Alchemy) in this tutorial because I have only committed checkpoints to git sub-modules which track the original repo. I was refraining from adding a dirty commit for these sub-modules for obvious reasons.
So pretty much this doc will be the only place where we would maintain this codebase. Thoughts?